x-pack/plugins/alerting/server/rules_client/common/wait_before_next_retry.ts

```diff
@@ -5,7 +5,7 @@
  * 2.0.
  */
 
-export const RETRY_IF_CONFLICTS_ATTEMPTS = 2;
+export const RETRY_IF_CONFLICTS_ATTEMPTS = 3;
```

Comment:
Curious to know what effect this change has on other parts of the rules client?

Reply:
@JiaweiWu it will try to solve 409 conflict (if it happens) maximum 3 time (not 2 as previously) for all ours bulk actions API and some other. Basically all endpoints which try to mutate rules saved object.